docs/_embed/partner_api.md

```diff
@@ -663,27 +663,27 @@ Additionally, if you’d prefer, you can open these urls in a separate window, n
   
 ### Prefill Options
 
-Always link the user with the `create_url` (available in a [Zap template object](/partner_api/endpoints#zap-template)) in order to create the Zap. Optionally, you can add additional parameters to the `create_url` so that the user's Zap is prefilled with the provided custom values.
+Prefills allow you to define the fields on behalf of the user, thus simplying the experience of setting up their Zap.
 
-> Note: You will need to know the fields that your app requires per step. You can find the fields as definied in your Zapier integration.
+You'll use the `create_url` (available in a [Zap template object](/partner_api/endpoints#zap-template)) in order to create the Zap. Next, you will add parameters to the `create_url` so that the user's Zap is prefilled with the provided custom values.
 
-Each parameter is in a flattened dictionary/object syntax. For example an object: `{a: {b: 2}}` would be flattened to: `a__b=2`. This allows you to provide countless prefills onbehalf of the user.
+> Note: You will need to know the fields that your app requires per step. You can find the fields as defined in your Zapier integration.
 
 **Example**
 
-Prefill Trello's board ID (field: `board`) in the second step of the Zap template:
+You can prefill the name of a Trello card (field: `name`) in the second step of the Zap template:
 
-`https://zapier.com/webintent/create-zap?template=2405?steps[1][params][board]=12345`
+`https://zapier.com/webintent/create-zap/?template=113&steps[1][params][name]=hello`
```

Comment:
With and without trailing slash works, and since we typically use without (https://docs.k8s.zapier.com/webintent/usage.html and also most code I've seen) doesn't use any, I'd drop it.